src/Extend/View.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Extension;
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Contracts\View\Factory;
+
+class View implements ExtenderInterface
+{
+    private $addNamespaces = [];
+    private $replaceNamespaces = [];
+
+    /**
+     * Register a new namespace of laravel views.
+     *
+     * @param  string  $namespace
+     * @param  string|array  $hints
```

Comment:
What does "hint" mean in this context? Isn't this a path (or multiple)?

Reply:
I just copied this from the laravel docblocks